Mathlib/RingTheory/Trace.lean

```diff
@@ -328,6 +328,56 @@ theorem Algebra.isIntegral_trace [FiniteDimensional L F] {x : F} (hx : IsIntegra
   · apply IsAlgClosed.splits_codomain
 #align algebra.is_integral_trace Algebra.isIntegral_trace
 
+lemma Algebra.trace_eq_of_algEquiv {A B C : Type*} [CommRing A] [CommRing B] [CommRing C]
+    [Algebra A B] [Algebra A C] (e : B ≃ₐ[A] C) (x) :
+    Algebra.trace A C (e x) = Algebra.trace A B x := by
```

Comment:
This could be used to golf [Algebra.discr_eq_discr_of_algEquiv](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Discriminant.html#Algebra.discr_eq_discr_of_algEquiv) and remove unnecessary assumptions.

Reply:
#9068